b2sdk/account_info/abstract.py

```diff
@@ -109,30 +110,42 @@ def clear_bucket_upload_data(self, bucket_id):
     @abstractmethod
     def get_account_id(self):
         """
-        Returns account_id or raises MissingAccountData exception
+        Returns account ID or raises MissingAccountData exception
 
         :rtype: str
         """
 
     @abstractmethod
+    def get_application_key_id(self):
+        """
+        Returns the application key ID used to authenticate
+
+        :rtype: str
+        """
+
+    @version_utils.rename_method(get_application_key_id, '0.1.5', '0.2.0')
```

Comment:
So clean, very nice.

Reply:
just curious do you have to include `get_account_id_or_app_key_id` the old method reference anywhere when doing this?